patches/instabug-reactnative+8.4.1.patch

```diff
@@ -11,6 +11,24 @@ index 09ec669..04e9e94 100644
    function setPromptOptionsEnabled(
      chat: boolean,
      bug: boolean,
+@@ -94,7 +94,7 @@ export namespace Replies {
+   function show(): void;
+   function setOnNewReplyReceivedCallback(onNewReplyReceivedCallback: () => void): void;
+   function setOnNewReplyReceivedHandler(onNewReplyReceivedHandler: () => void): void;
+-  function getUnreadRepliesCount(messageCountCallback: () => void): void;
++  function getUnreadRepliesCount(messageCountCallback: (count: number) => void): void;
+   function setInAppNotificationsEnabled(inAppNotificationsEnabled: boolean): void;
+   function setInAppNotificationSound(shouldPlaySound: boolean): void;
+ }
+@@ -140,7 +140,7 @@ export function setIBGLogPrintsToConsole(printsToConsole: boolean): void;
+ export function setCrashReportingEnabled(enableCrashReporter: boolean): void;
+ export function setDidSelectPromptOptionHandler(didSelectPromptOptionHandler: () => void): void;
+ export function setSessionProfilerEnabled(sessionProfilerEnabled: boolean): void;
+-export function getUnreadMessagesCount(messageCountCallback: () => void): void;
++export function getUnreadMessagesCount(): number;
```

Comment:
```suggestion
+export function getUnreadMessagesCount((count : number) => void): void;
```
it works just like that _**messageCountCallback**_
https://github.com/Instabug/Instabug-React-Native/blob/6d84e793acb0111bbec91d8bad223268e12d4407/android/src/main/java/com/instabug/reactlibrary/RNInstabugReactnativeModule.java#L726